src/transport_none.h

```diff
@@ -311,4 +311,34 @@ void shmem_transport_syncmem(void)
     return;
 }
 
+static inline
+uint64_t shmem_transport_get_pending_put_cntr(shmem_transport_ctx_t *ctx)
+{
+    return -1;
```

Comment:
These functions return an unsigned quantity, so 0 might be a better choice than -1.  In fact, 0 is a good choice since there is no transport and therefore these counters are effectively stuck at 0.